modules/ghlint-snakeyaml/src/main/kotlin/net/twisterrob/ghlint/model/SnakePermissions.kt

```diff
@@ -0,0 +1,51 @@
+package net.twisterrob.ghlint.model
+
+import net.twisterrob.ghlint.yaml.getOptionalText
+import org.snakeyaml.engine.v2.nodes.MappingNode
+
+public class SnakePermissions internal constructor(
+		internal val node: MappingNode,
+		override val map: Map<String, String>
+) : Permissions {
+	override val actions: Access
+		get() = node.getOptionalText("actions")?.let { Access.fromString(it) } ?: Access.NONE
```

Comment:
```suggestion
	override val actions: Access get() = node.access(Permission.ACTIONS)
```
with `private access()` at the bottom of the file &rarr; immediate 30% of file size + easier to change. It also removes constant duplication.